templates/artist/includes/artist_detail_overview.html

```diff
@@ -4,140 +4,125 @@
-                <input type="number" min="1" max="{{ campaign.num_shares_remaining }}" value="{{ campaign.default_num_shares }}" />
-                <button id="add-shares" class="secondary hollow button" type="button">+</button>
-            </div>
-            <div class="total-cost">
-                <h4 id="subtotal-shares-price"></h4>
-                <span data-tooltip aria-haspopup="true" class="has-tip [tip-right] [radius round]" title="${{ PERDIEM_FEE }} PerDiem + Credit Card Fee (2.9%+$0.30)">
-                    <h4 id="fees-price"></h4>
-                </span>
-                <h4 id="shares-price"></h4>
-            </div>
-            <button id="invest-button" class="button" type="button">Buy {{ campaign.default_name_shares }} Share{{ campaign.default_num_shares|pluralize }}</button>
-        {% else %}
-            <p><a data-open="login-modal">Login</a> to invest in {{ artist.name }}.</p>
-        {% endif %}
-        <br />
-        <br />
-    {% endif %}
-    {% if artist.bio %}
+        <h4>AMOUNT RAISED</h4>
+        <h4 class="investing">${{ campaign.amount_raised|intcomma }}</h4>
+        <h5>OUT OF ${{ campaign.amount|intcomma }}</h5>
         <div class="break">
             <hr>
         </div>
-        <!-- Artist Bio -->
-        <h2>OUR STORY</h2>
-        <div class="artist-detail-padding">
-            {{ artist.bio.bio|markdown }}
-        </div>
-    {% endif %}
 
-    {% if artist.soundcloudplaylist_set.exists %}
+            {% if request.user.is_authenticated %}
+                <!-- Buy Shares -->
+                <button id="invest-button" class="button buy" type="button">Buy {{ campaign.default_name_shares }} Share{{ campaign.default_num_shares|pluralize }}</button>
+                <div class="invest-num-shares">
+                    <button id="remove-shares" class="secondary hollow button disabled shares" type="button" disabled>-</button>
+                    <input type="number" min="1" max="{{ campaign.num_shares_remaining }}" value="{{ campaign.default_num_shares }}" />
+                    <button id="add-shares" class="secondary hollow button" type="button">+</button>
+                </div>
+                <p>NUMBER OF SHARES</p> 
+                <div class="total-cost">
+                    <span data-tooltip aria-haspopup="true" class="has-tip [tip-right] [radius round]" title="${{ PERDIEM_FEE }} PerDiem + Credit Card Fee (2.9%+$0.30)">
+                        Total <a id="shares-price"></a>
+                    </span>
+                </div>
+            {% else %}
+                <h4> to invest in this project</h4>
+                <button class="login-box signup" data-open="login-modal">Login</button>
+                <button class="login-box signup" >Sign up</button>
+            {% endif %}
+    </div>
+    {% endwith %}
+
+        <!-- Music -->
+        {% if artist.soundcloudplaylist_set.exists %}
```

Comment:
This `{% if %}` block should just surround the Soundcloud playlist. Otherwise the genres and artist location will not show if they don't have any music to preview.